server/src/main/java/org/opentosca/toscana/plugins/kubernetes/KubernetesPlugin.java

```diff
@@ -39,63 +30,15 @@ private static Platform getPlatformDetails() {
     @Override
     public void transform(TransformationContext context) throws Exception {
```

Comment:
Please Do not override this method, its already implemented by the lifecycle aware plugin

Reply:
@c-mueller maybe then this needs to become `final`?

Reply:
Theoretially, would not force it though.